test/lib/sstable_test_env.hh

```diff
@@ -14,6 +14,8 @@
 #include <seastar/util/defer.hh>
 
 #include "sstables/sstables.hh"
+#include "compaction/compaction_manager.hh"
```

Comment:
nit: commitlog says "start a compaction..." but I don't see anything started here, just created.

Reply:
It's a degenerate start since it's a single shard that's constructed with `compaction_manager(compaction_manager::for_testing_tag{})`.
I'll reword the commit message to make it clearer.